boa/src/object/mod.rs

```diff
@@ -99,7 +112,137 @@ pub enum ObjectData {
     NativeObject(Box<dyn NativeObject>),
 }
 
-impl Display for ObjectData {
+impl ObjectData {
```

Comment:
I think we should add documentation to all these methods. It's pretty straightforward, but still, something saying what each creates.